packages/workbox-background-sync/src/lib/QueueStore.ts

```diff
@@ -7,27 +7,8 @@
 */
 
 import {assert} from 'workbox-core/_private/assert.js';
-import {DBWrapper} from 'workbox-core/_private/DBWrapper.js';
-import {RequestData} from './StorableRequest.js';
 import '../_version.js';
-
-
-const DB_VERSION = 3;
-const DB_NAME = 'workbox-background-sync';
-const OBJECT_STORE_NAME = 'requests';
-const INDEXED_PROP = 'queueName';
-
-export interface UnidentifiedQueueStoreEntry {
-  requestData: RequestData;
-  timestamp: number;
-  id?: number;
-  queueName?: string;
-  metadata?: object;
-}
-
-export interface QueueStoreEntry extends UnidentifiedQueueStoreEntry {
-  id: number;
-}
+import {UnidentifiedQueueStoreEntry, QueueStoreEntry, QueueDb} from './QueueDb';
```

Comment:
Please add the `.js` suffix.

Reply:
We should find a way to add this to our lint :)